### ISSUE TITLE
Nested fields of dictionaries, many=True

### ISSUE THREAD

**Issue by writeson** [marshmallow-code/marshmallow#726]
Hi all,
I'm using Marshmallow to serialize data structures prior to sending them down the wire as JSON strings for a ReSTful interface. One of the structures I'm trying to serialize looks like this:
```
{
    'thing': 'thing text',
    'count': 1,
    'table_data': {
        'name_1': { ... },
        'name_2': {....}
    }
}
```
The table_data field is an iterable, but it's a dictionary, not a list, and the dictionaries all are structured the same. I'd like to create schema's for this, but searching around the docs and with Google it looks like this isn't supported. I'd like to be able to create schema's for the repeating dictionary structures. Can this be added?

Thanks in advance,
Doug

**Comment by lafrech** [comment]
I believe what you're looking for was added to Marshmallow just recently.

See [SO question](https://stackoverflow.com/q/38048775/4653485), [issue](https://github.com/marshmallow-code/marshmallow/issues/483) and [PR](https://github.com/marshmallow-code/marshmallow/pull/700).

It is in Marshmallow v3 branch. AFAIK, the release date of Marshmallow 3 is... when it is ready.

You may want to give a try to the beta versions. Beware of the breaking changes. Read the _upgrading_ section in the docs.

**Comment by writeson** [comment]
Jerome,

Thanks for the heads up, I'll take a look.

Doug

On Thu, Jan 18, 2018 at 3:42 PM, Jérôme Lafréchoux <notifications@github.com

**Comment by deckar01** [comment]
Duplicate of #483.

It was recently implemented by #700. It will be part of the 3.0 release.
